src/main/java/seedu/financialplanner/cashflow/Budget.java

```diff
@@ -1,11 +1,15 @@
 package seedu.financialplanner.cashflow;
 
 import java.text.DecimalFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static seedu.financialplanner.FinancialPlanner.currentDate;
 
 public abstract class Budget {
-    private static final int MONTH = 30;
     private static double initialBudget = 0;
     private static double currentBudget = 0;
+    private static LocalDate date = currentDate;
```

Comment:
can consider using LocalDate.now instead of using public currentDate from the financialplanner